code/modules/admin/topic.dm

```diff
@@ -1395,19 +1395,33 @@
 			to_chat(usr, "This can only be used on instances of type /mob/living/carbon/human.")
 			return
 
-		var/obj/item/reagent_containers/food/snacks/cookie/cookie = new(H)
-		if(H.put_in_hands(cookie))
+		//if(is_species(H, /datum/species/lizard))
+		
+		//let's keep it simple
+		//milk to plasmemes and skeletons, meat to lizards, electricity bars to ethereals, cookies to everyone else
+		//var/obj/item/reagent_containers/food/snacks/cookie/cookie = new(H)
+		var/cookiealt = /obj/item/reagent_containers/food/snacks/cookie
+		if(is_species(H, /datum/species/skeleton))
+			cookiealt = /obj/item/reagent_containers/food/condiment/milk
+		if(is_species(H, /datum/species/plasmaman))
+			cookiealt = /obj/item/reagent_containers/food/condiment/milk
+		if(is_species(H, /datum/species/ethereal))
+			cookiealt = /obj/item/reagent_containers/food/snacks/energybar
+		if(is_species(H, /datum/species/lizard))
+			cookiealt = /obj/item/reagent_containers/food/snacks/meat/slab

```

Comment:
Use these defines https://github.com/tgstation/tgstation/blob/master/code/__DEFINES/is_helpers.dm#L58

Reply:
Actually this can be done with a switch
```
switch(H?.dna.species)
	if("skeleton")
		cookiealt = /obj/item/reagent_containers/food/condiment/milk
	if("plasmaman")
		cookiealt = /obj/item/reagent_containers/food/condiment/milk
	if("ethereal")
		cookiealt = /obj/item/reagent_containers/food/snacks/energybar
	if("lizard")
		cookiealt = /obj/item/reagent_containers/food/snacks/meat/slab
```

Reply:
This switch looks extremely prone to errors due to maintenance. This should probably linked to type in some way instead of name.

Reply:
\>Maintainability over negligible performance micro-optimisations
what a nerd

Use an else-if chain at least.

Reply:
done

Reply:
oh i didnt reload the page shit

Reply:
done